docs/range-proof-protocol.md

```diff
@@ -1,190 +1,315 @@
-The `range_proof` module contains API for producing a range proof for a given integer value.
+The `aggregated_range_proof` module contains an API for producing a proof that multiple integer values are within a certain range.
 
-Range proof protocol
-====================
+Aggregated proof protocol
+=========================
 
-This is a documentation for the internal implementation of a range proof. 
-You may find the introduction to all the pieces of the protocol in the [notes](../notes/index.html) module.
+This is a documentation for the internal implementation of the aggregated range proof. You may find the introduction to all the pieces of the range proof protocol in the [notes](../notes/index.html) module.
 
-The range proof is a zero-knowledge proof of the following relation
+The aggregated range proof is a zero-knowledge proof of the following relation:
 \\[
 \operatorname{ZK-PK}\left\\{
-  v \in {\mathbb Z\_p}
-  : v \in [0, 2^n)
+  v_{(j)} \in {\mathbb Z\_p}
+  : v_{(j)} \in [0, 2^n)
```

Comment:
I think this might be clearer if it were written as `v_{(1)}, \ldots, v_{(m)}` since that reinforces that there are multiple values and also introduces the `m` parameter up front in the problem statement as the number of values.

Reply:
👍